webapp/client/src/pages/PauschbetragPersonAPage.js

```diff
@@ -0,0 +1,95 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { Trans, useTranslation } from "react-i18next";
+import styled from "styled-components";
+import FormHeader from "../components/FormHeader";
+import StepForm from "../components/StepForm";
+import StepHeaderButtons from "../components/StepHeaderButtons";
+import { selectionFieldPropType } from "../lib/propTypes";
+import FormFieldRadio, { boldifyChoices } from "../components/FormFieldRadio";
+import Details from "../components/Details";
+
+const DetailsDiv = styled.div`
+  margin-bottom: var(--spacing-04);
+`;
+
+export default function PauschbetragPersonAPage({
+  stepHeader,
+  form,
+  fields,
+  prevUrl,
+}) {
+  const { t } = useTranslation();
+  const boldChoices = boldifyChoices(fields.personAWantsPauschbetrag.options);
+
+  return (
+    <>
+      <StepHeaderButtons url={prevUrl} />
+      <FormHeader
+        title={stepHeader.title}
+        intro={
+          <Trans
+            t={t}
+            i18nKey="lotse.pauschbetrag.intro"
+            components={{
+              bold: <b />,
+            }}
+          />
+        }
+      />
+      <DetailsDiv>
+        <Details
+          title={t("lotse.pauschbetrag.details.title")}
+          detailsId="pauschbetrag_person_a_details"
+        >
+          {[
+            <p key={0}>{t("lotse.pauschbetrag.details.p1")}</p>,
+            <ul key={1}>
+              <li key={0}>{t("lotse.pauschbetrag.details.list.listItem1")}</li>
+              <li key={1}>{t("lotse.pauschbetrag.details.list.listItem2")}</li>
+              <li key={2}>{t("lotse.pauschbetrag.details.list.listItem3")}</li>
+            </ul>,
+            <p key={2}>{t("lotse.pauschbetrag.details.p2")}</p>,
+            <p key={3}>
+              <Trans
+                t={t}
+                i18nKey="lotse.pauschbetrag.details.p3"
+                components={{
+                  bold: <b />,
+                }}
+              />
+            </p>,
+            <p key={4}>{t("lotse.pauschbetrag.details.p4")}</p>,
+          ]}
+        </Details>
+      </DetailsDiv>
+      <StepForm {...form}>
+        <FormFieldRadio
+          fieldName="person_a_wants_pauschbetrag"
+          fieldId="person_a_wants_pauschbetrag"
+          label={{ text: "" }}
+          options={boldChoices}
```

Comment:
Okay, so the choice values should be set in the backend. And I think this is also the reason I did it: The backend has to know what the choices are to be able to validate that a valid choice is set. That said, the backend does not have to set the texts. We could do that in the frontend. But I couldn't make out a way to reliably set the correct frontend texts to the correct backend values. I'd have to duplicate the values (as you're suggesting in your comment) and I'm somehow reluctant to do so.

Reply:
We talked about it and I realized that there is not even the possibility to completely set it in the backend because of the bold-part of the problem. So your solution it is^^